lisa/nic.py

```diff
@@ -158,6 +158,14 @@ def get_lower_nics(self) -> List[str]:
     def get_device_slots(self) -> List[str]:
         return [x.pci_slot for x in self.nics.values() if x.pci_slot]
 
+    def get_current_device_driver(self, nic: NicInfo) -> str:
```

Comment:
1. Pass in the nic_name is easier to use.
2. the name can be simple like `get_driver`.